common/lib/capa/capa/util.py

```diff
@@ -195,3 +195,10 @@ def get_inner_html_from_xpath(xpath_node):
     # strips outer tag from html string
     inner_html = re.sub('(?ms)<%s[^>]*>(.*)</%s>' % (xpath_node.tag, xpath_node.tag), '\\1', html)
     return inner_html.strip()
+
+
+def remove_markup(html):
+    """
+    Return the html without markup.
+    """
+    return bleach.clean(html, tags=[], strip=True)
```

Comment:
@robrap Even with `HTML(bleach.clean(html, tags=[], strip=True))` html is escaped twice. Please see the [gist](https://gist.github.com/muhammad-ammar/c712500596df62d917973f6e40d53608)